HW7/phone_book.py

```diff
@@ -0,0 +1,65 @@
+phone_book = {}
+
+# Додавання контакту
+def add_contact():
```

Comment:
when function is used - better to write comments inside function
```suggestion
def add_contact():
    """
    Додавання контакту до телефонної книги
    """
```